Remove `poststart` script

**Issues:**
- On shutting down of the `npm start` script, you'll receive the following error:
```
npm ERR! code ELIFECYCLEform) 
npm ERR! errno 2
npm ERR! sparkpress@1.0.0 docker: `docker-compose up`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the sparkpress@1.0.0 docker script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ricardo/.npm/_logs/2021-03-31T15_55_59_665Z-debug.log
```
and just hangs.

**Proposed Solution**
Remove `poststart` in favor of `"docker": "docker-compose up --abort-on-container-exit",`

--- a/package.json
+++ b/package.json
@@ -6,7 +6,6 @@
   "scripts": {
     "prestart": "run-p clean node-version",
     "start": "run-p build:dev serve:dev",
-    "postserve:dev": "docker-compose down",
     "clean": "rm -rf theme/ plugins/example-blocks/",
     "copy": "run-p copy:*",
     "copy:php": "mkdir -p ./theme && cp -r ./src/php/* ./theme",
@@ -15,7 +14,7 @@
     "prebuild:prod": "npm run clean",
     "build:dev": "run-p plugins:dev copy:php sass watch js:watch",
     "build:prod": "run-s plugins:build copy js:prod 'sass -- --style compressed'",
-    "serve:dev": "docker-compose up",
+    "serve:dev": "docker compose up --abort-on-container-exit || docker compose down",
     "sass": "sass src/scss/base.scss theme/css/base.css",
     "postsass": "postcss theme/css/base.css --use autoprefixer -r",
     "js:watch": "NODE_ENV=development node esbuild.mjs",

